database-population.js

```diff
@@ -0,0 +1,72 @@
+const faker = require('faker');
+const db = require('./database/reviews');
+
+const numUsers = 100;
+const numReviews = 500;
+
+const arrayOfAvatarURLs = ['https://s3-us-west-1.amazonaws.com/fec-reviews/1.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/2.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/3.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/4.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/5.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/6.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/7.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/8.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/9.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/10.jpg'];
+
+let arrayOfUsers = [];
+
+const generateRandomNumber = function (val) {
+    return Math.ceil(val * Math.random());
+}
+
+const userGeneration = function () {
+    for (let i = 0; i < numUsers; i++) {
+        let tempObject = {};
+        tempObject.userId = i;
+        tempObject.name = faker.name.findName();
+        tempObject.avgRating = generateRandomNumber(5);
+        // will break in the case where Math.random() returns 1 - not sure if - 0.01 will work
+        tempObject.picture = arrayOfAvatarURLs[Math.floor((arrayOfAvatarURLs.length - 0.01) * Math.random())];
```

Comment:
Math.random() won't return 1 ever, so I don't think you have to worry about that case. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)

Also, wondering if you should use your own 'generateRandomNumber' here?